src/graphql/mutation/createSavedSearch.graphql

```diff
@@ -0,0 +1,15 @@
+mutation createSavedSearch($name: String!, $filters: LoanSearchFiltersInput) {
+  my {
+    createSavedSearch (savedSearch :{
+      filters: $filters,
+      name: $name,
+      isAlert: false,
+      queryString: $name,
```

Comment:
The queryString param should contain the current query params from the page, not including any `utm_` params.

It looks like the url that we use in the query presentation will be auto generated from the filters in the LSC.

Reply:
gotcha, just to confirm, i'm fixing it so that the `queryString` String value looks as such (pulled from the url query params) -

`{"gender":"female","country":"PH"}`

not sure if there is a way to verify this, the graphql schema just indicates it should be a string

Reply:
I think it's the url format for this one. You could test which form works best by passing the value to a lend > loams query with only the queryString filter applied.

That said, it may turn out to be something even more specifically formatted for php. We can check the monologue code in a little while